truss/truss/config.go

```diff
@@ -0,0 +1,41 @@
+package truss
+
+import (
+	"path/filepath"
+)
+
+// Config defines the inputs to a truss service generation
+type Config struct {
+	// The first path in $GOPATH
+	GOPATH string
+
+	// The path to where .pb.go files protoc-gen-go creates will be written
```

Comment:
I like that idea. Although what happens when the GOPATH is composed of multiple paths?